.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Issue template
+about: 'Issue template for cloudnative-pg.io '
+title: Issue cloudnative-pg website
```

Comment:
comparing with the operator repo issue template, it seems there the `description` field is the one where we'd want this, and the `title` field is the prefix to denote new issues, like `[Doc]: `